### PR TITLE
Document Basic Usage of the WeeblyClient

### PR DESCRIPTION
**Pull request by kiastu**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,3 +1,53 @@
...
+### Instantiating the Client
+
+You may insantiate a new Weebly Client as such:
+
+```
+$client = \Weebly\WeeblyClient($client_id, $client_secret, $user_id, $site_id, $access_token);
+```
+
+The only mandatory options that you must provide to the constructor are `$client_id` and `$client_secret`. You can find them both in your dev dashboard at <https://www.weebly.com/developer-admin> once you have registered a developer account with Weebly.
+Please store these credentials securely on your server, and not expose them publicly.
```
Comment (robinwhitmore) [inline readme.md]: s/b "...and do not expose them publically.

File: readme.md [hunk 2]
```diff
@@ -1,3 +1,53 @@
...
+The `$user_id`, `site_id` and `$access_token` are made accessible to you when a user grants your app permissions through the App OAuth flow. You do not need to provide these parameters unless you are accessing the OAuth methods. (i.e. _*Store APIs*_, _*Site APIs*_)
+You can find more information about the OAuth flow and how to obtain the `$user_id`, `site_id` and `$access_token` at <https://dev.weebly.com/configure-oauth.html>.
+
+### Authorizing through the OAuth Flow
+The WeeblyClient provides a few helper methods to aid you through the OAuth flow.
+    * `getAuthorizationUrl` Returns a properly formatted authorization URL that you should respond with after our servers initiate the OAuth flow.
+    * `getAccessToken` :Retrieves the access token once you have obtained the necessary authorization code in OAuth.
```
Comment (robinwhitmore) [inline readme.md]: Either use the ":" after the call or don't, but pick one. ;-)

File: readme.md [hunk 3]
```diff
@@ -1,3 +1,53 @@
...
+
+You can find more information about the Weebly OAuth flow at <https://dev.weebly.com/configure-oauth.html>.
```
Comment (robinwhitmore) [inline readme.md]: You may want to consider pointing them to <https://dev.weebly.com/authentication-with-oauth2.html> first, so they understand the flow. It's kind of backwards from most OAuth flows so if they go right into config, they might get confused.

File: readme.md [hunk 4]
```diff
@@ -1,3 +1,55 @@
...
+### Instantiating the Client
+
+You may insantiate a new Weebly Client as such:
+
+```php
+$client = \Weebly\WeeblyClient($client_id, $client_secret, $user_id, $site_id, $access_token);
+```
+
+The only mandatory options that you must provide to the constructor are `$client_id` and `$client_secret`. You can find them both in your dev dashboard at <https://www.weebly.com/developer-admin> once you have registered a developer account with Weebly.
+Please store these credentials securely on your server, and do not expose them publically.
```
Comment (puddi) [inline readme.md]: publically isn't a word.

File: readme.md [hunk 5]
```diff
@@ -1,3 +1,55 @@
...
+
+```php
+$endpoint = '/user/sites/' . $site_id;
+$response = $client.get($endpoint);
+```
+
+Mark a product as shipped:
+```php
+$endpoint = '/user/sites/' . $site_id . '/store/orders/' . $order_id . '/shipments/' . $order_shipment_id;
+$product = ["tracking_number"=> "1234567810abcde"]; //Modifying the tracking_number on an unshipped product will mark it as shipped as well!
```
Comment (puddi) [inline readme.md]: should be:

```php
$product = ['tracking_number' => '12345678910abcde']; // Modifying the tracking_number on an unshipped product will mark it shipped as well!
```

**Comment by puddi** [comment]
:+1: otherwise

File: readme.md (hunk 4 above)
Comment (kiastu) [inline readme.md]: I googled it, it actually is ! "Lawyers like using bigger words"

**Pull request merged**